src/main/resources/application.yml

```diff
@@ -0,0 +1,25 @@
+management:
+  security:
+    enabled: false
+
+cgi:
+  jms:
+    destination.name: CP_OutboundQueue
+
+spring:
+  artemis:
+    mode: native
+    host: ${jms.host:localhost}
+    user: ${jms.username:jmsuser}
+    password: ${jms.password:jmsuser}
+    port: 5445
+
+#  # Spring JMS Settings
```

Comment:
Do we need these commented values?

Reply:
No - you are right - have got rid of all of them